### ISSUE TITLE
probable with getting the plugin to get "check_state.js"

### ISSUE THREAD

**Issue by marcfolio** [apexad/homebridge-garagedoor-command#23]
Hi, this plugin is everything I've been looking for. And it's working fantastic so far except for getting the state of the door. 

My setup is a raspberry pi zero w with a relay attached, and a door switch (the magnet attached to the door and switch attached to the ground). 

I can successfully get the plugin to open & close using a "close.sh" to execute a python script that triggers the relay.

For the state though I'm having troubles. I've tried multiple things (written in python and node.js). But they always break when state is polled. Since your documentation mentions "check_state.js" this is the JS code I've written:

const Gpio = require('onoff').Gpio;
const button = new Gpio(16, 'in', 'both');
var _state;

```
button.watch((err, value) => {
  
  if(value == "0"){
  	_state = "OPEN"
  }else{
  	_state = "CLOSED"
  }
  console.log( _state);
});
```

The error I get when I put "check_state.js" in the config file is: 
```[5/30/2020, 16:16:35] [Garage Door] Error: Error: Command failed: /var/lib/homebridge/scripts/check_state.js
/var/lib/homebridge/scripts/check_state.js: 1: /var/lib/homebridge/scripts/check_state.js: Syntax error: "(" unexpected```

Any help you can give would be great! Thank you!

**Comment by apexad** [comment]
I've mentioned this before.  You may want to consider https://github.com/benlamonica/homebridge-rasppi-gpio-garagedoor

Otherwise, the scripts can be python, NodeJS, or bash.  Your javascript looks fine.  Are you calling with `node check_state.js`?

**Comment by marcfolio** [comment]
I looked at that plugin and tried installing it, but I don't think it was up-to-date and I had issues. Seriously your plugin is the best I've seen so far (and fairly easy to extend). Still I can't get node and check_state.js to work, BUT I was successful in getting python to run. 

With the Node.js what's the best way to return "CLOSED" or "OPEN"? I've tried `console.log`, `return`, but nothing seems to work. Would love to know if you have a solution?

AS for python... this might help someone in the future:

This is the Plugin CONFIG setup:
```
"name": "Garage Door",
 "open": "/var/lib/homebridge/scripts/open.sh",
"close": "/var/lib/homebridge/scripts/close.sh",
"state": "/var/lib/homebridge/scripts/state.sh",
"status_update_delay": "15",
"poll_state_delay": 60,
"accessory": "GarageCommand"

The "open" and "close" are essentially same code (you could reuse the same .sh and .py if you wanted).

open.sh
```
python3 /var/lib/homebridge/scripts/open.py
```

open.py
```
from gpiozero import LED
from time import sleep

relay = LED(26)

relay.on()
sleep(1)
relay.off()
```

state.sh
```
python3 /var/lib/homebridge/scripts/state.py
```

state.py 
```
from gpiozero import Button

button = Button(2)

if button.is_pressed:
   print('CLOSED')
else:
   print('OPEN')
```
